sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -343,6 +343,31 @@ def delete_container(
         except StorageErrorException as error:
             process_storage_error(error)
 
+    @distributed_trace
+    def undelete_container(self, deleted_container_name, deleted_container_version, **kwargs):
+        # type: (str, str, **Any) -> None
+        """Restores soft-deleted container.
+
+        Operation will only be successful if used within the specified number of days
+        set in the delete retention policy.
+
+        :param str deleted_container_name:
+            Specifies the name of the deleted container to restore.
+            Servivce Version 2019-12-12 and laster.
+        :param str deleted_container_version:
+            Specifies the version of the deleted container to restore.
+            Servivce Version 2019-12-12 and laster.
```

Comment:
Typo: "later"
We should also add the docstring tag for which SDK version this was introduced in.